src/Stratis.Features.FederatedPeg/FederatedPegOptions.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NBitcoin;
+
+namespace Stratis.Features.FederatedPeg
+{
+    public class FederatedPegOptions
+    {
+        public Network CounterChainNetwork { get; }
```

Comment:
Not sure if this is suitable but perhaps a comment here to explain what the counter chain network is? I know it might be obvious but to a new dev it might not be?